Shared/Samples/Change map view background/README.md

```diff
@@ -0,0 +1,36 @@
+# Change map view background
+
+Customize map view's background by changing its grid properties.
+
+| ChangeMapViewBackground | Settings |
+| ----------------- | ------------- |
+|![Image of change map view background](change-map-view-background.png)|![Image of change map view background](change-map-view-background-settings.png)|
+
+## Use case
+
+Basemaps should be selected contextually. For example, in maritime applications, it would be more appropriate to use a basemap of the world's oceans as opposed to a basemap of the world's streets.
```

Comment:
I don't think this is a proper use case description. When I read it, I'm expecting a sample that shows how to select a basemap style such as the [Set basemap](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/main/Shared/Samples/Set%20basemap) sample, that uses an `arcGISOceans` style.

Can you help reword this base on the functionalities of the sample? Thanks.